plugins/modules/zabbix_template.py

```diff
@@ -429,9 +462,17 @@ def check_template_changed(self, template_ids, template_groups, link_templates,
             if template_macros != existing_macros:
                 changed = True
 
+        if LooseVersion(self._zbx_api_version) >= LooseVersion('4.2'):
+            if 'tags' not in existing_template['zabbix_export']['templates'][0]:
+                existing_template['zabbix_export']['templates'][0]['tags'] = []
+            if template_tags is not None:
+                existing_tags = existing_template['zabbix_export']['templates'][0]['tags']
+                if template_tags != existing_tags:
```

Comment:
I wonder if we should not rather utilize helper function for comparing list of dictionaries from module_utils/

Reply:
I agree, will try it later. Initially I've used the same handling as for macros.

Reply:
Well, I tried to use helpers, but the code was less readable IMO and the result was the same. If its working fine using just simple compare, than I would leave it as it. I think helpers can be useful for some more complex cases.

Reply:
ok, in that case I have no further objections :)